packages/web-components/fast-html/src/components/element.ts

```diff
@@ -0,0 +1,27 @@
+import { attr, FASTElement, Observable } from "@microsoft/fast-element";
+
+export abstract class RenderableFASTElement extends FASTElement {
+    @attr({ mode: "boolean", attribute: "defer-hydration" })
+    deferHydration: boolean = true;
+
+    @attr({ mode: "boolean", attribute: "needs-hydration" })
+    needsHydration: boolean = true;
+
+    constructor() {
+        super();
+
+        this.setAttribute("defer-hydration", "");
+        this.setAttribute("needs-hydration", "");
+
+        Observable.defineProperty(this.$fastController.definition, "shadowOptions");
```

Comment:
should we unsubscribe after deferred?

Reply:
It shouldn't need to since it's setting a single attribute which should never get set again, and in this case it would actually be a perf hit to unsubscribe.